src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rent = 40 * days;
+
+  if (days >= 3 && days < 7) {
+    rent -= 20;
```

Comment:
5 point in [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)